docs/source/code/spatialstats.py

```diff
@@ -0,0 +1,40 @@
+import xdem
+import geoutils as gu
+import numpy as np
+
+# load diff and mask
+xdem.examples.download_longyearbyen_examples(overwrite=False)
+
+reference_raster = gu.georaster.Raster(xdem.examples.FILEPATHS["longyearbyen_ref_dem"])
+to_be_aligned_raster = gu.georaster.Raster(xdem.examples.FILEPATHS["longyearbyen_tba_dem"])
+glacier_mask = gu.geovector.Vector(xdem.examples.FILEPATHS["longyearbyen_glacier_outlines"])
+inlier_mask = ~glacier_mask.create_mask(reference_raster)
+
+nuth_kaab = xdem.coreg.NuthKaab()
+nuth_kaab.fit(reference_raster.data, to_be_aligned_raster.data,
+              inlier_mask=inlier_mask, transform=reference_raster.transform)
+aligned_raster = nuth_kaab.apply(to_be_aligned_raster.data, transform=reference_raster.transform)
+
+ddem = gu.Raster.from_array((reference_raster.data - aligned_raster),
```

Comment:
Now you can just take the difference of both rasters, without using this convoluted method!

Reply:
Right, but aligned_raster is not a Raster, just an array. Never mind.